src/Application/Actions/Donations/Confirm.php

```diff
@@ -92,6 +92,17 @@ protected function action(Request $request, Response $response, array $args): Re
             throw new NotFoundException();
         }
 
+        if (trim($paymentMethodId) === '') {
+            $donationID = $donation->getId();
+            $this->logger->error(
+                <<<EOF
+Donation Confirmation attempted with blank payment method id "$paymentMethodId" for Donation #$donationID 
```

Comment:
UUID is surfaced in more places so maybe more helpful to log instead / as well?

Reply:
Makes sense, I will change it.

Reply:
Done in 6900cdf